app/models/config.rb

```diff
@@ -12,21 +12,22 @@ class Config < ApplicationRecord
     practical_support_guidance_url: 'A link to a Google Drive folder with Practical Support resources. ' \
                    'Ex: https://drive.google.com/drive/my-practical_support',
     fax_service: 'A link to your fax service. ex: https://www.efax.com',
-    start_of_week: "How to render your budget bar. Default is weekly starting on Monday. Enter \"Sunday\" for weekly budget starting on Sunday, or \"Monthly\" for a calendar month based budget.",
-    budget_bar_max: "The maximum for the budget bar. Defaults to 1000 if not set. Enter as a number with no dollar sign or commas.",
+    start_of_week: 'How to render your budget bar. Default is weekly starting on Monday. Enter "Sunday" for weekly budget starting on Sunday, or "Monthly" for a calendar month based budget.',
```

Comment:
I set up a new machine recently and it had autoformat on save set up. mad annoying! but I'm gonna just let this sit for now